Source/WebCore/platform/ios/PlatformScreenIOS.mm

```diff
@@ -82,6 +82,9 @@ bool screenSupportsExtendedColor(Widget*)
 bool screenSupportsHighDynamicRange(Widget*)
 {
 #if USE(MEDIATOOLBOX)
+    if (auto data = screenData(primaryScreenDisplayID()))
+        return data->screenSupportsHighDynamicRange;
```

Comment:
This should be moved outside the `#if USE` macro block.

Reply:
Done!

Thanks for reviewing!